packages/multichain-network-controller/src/types.ts

```diff
@@ -117,6 +132,15 @@ export type MultichainNetworkControllerSetActiveNetworkAction = {
   handler: SetActiveNetworkMethod;
 };
 
+export type GetNetworksWithTransactionActivityByAccountsMethod =
+  () => Promise<ActiveNetworksByAddress>;
+
+export type MultichainNetworkControllerGetNetworksWithTransactionActivityByAccountsAction =
+  {
+    type: `${typeof MULTICHAIN_NETWORK_CONTROLLER_NAME}:getNetworksWithTransactionActivityByAccounts`;
+    handler: GetNetworksWithTransactionActivityByAccountsMethod;
+  };
```

Comment:
Nit: Would it make sense to use `MultichainNetworkController['getNetworksWithTransactionActivityByAccounts']`?

``` suggestion
export type MultichainNetworkControllerGetNetworksWithTransactionActivityByAccountsAction =
  {
    type: `${typeof MULTICHAIN_NETWORK_CONTROLLER_NAME}:getNetworksWithTransactionActivityByAccounts`;
    handler: MultichainNetworkController['getNetworksWithTransactionActivityByAccounts'];
  };
```